plugins/hls-call-hierarchy-plugin/README.md

```diff
@@ -0,0 +1,23 @@
+# Call hierarchy plugin for the [Haskell Language Server](https://github.com/haskell/haskell-language-server#readme)
+
+The call hierarchy plugin can review the code to determine where functions are called and how they relate to other functions.
+
+This plugin is useful when debugging and refactoring code because it allows you to see how different parts of the code are related. And it is more conducive for users to quickly understand their macro architecture in the face of strange code.
+
+## Demo
+
+![Call Hierarchy in Emacs](call-hierarchy-in-emacs.gif)
+
+![Call Hierarchy in VSCode](call-hierarchy-in-vscode.gif)
+
+## Prerequisite
+None. You can experience the whole feature without any setting.
+
+## Configuration
+None. Native support by lsp-based client, switch on and off is insignificant.
```

Comment:
i would add the basic configuration to enable/disable the plugin (in json format copypasteable). With the value enabled a true of course (the default value)